autoversioner/tagger/client.go

```diff
@@ -0,0 +1,79 @@
+package tagger
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+
+)
+
+const (
+	botUserName = "guesttestinfra-bot"
```

Comment:
can we pass these in instead of making them constants? that way updates are just to config rather than to code, you can run testing versions, etc.

Reply:
sure